### DISCUSSION TITLE
Cannot Set CSS Styles for Specific Cells in SlickGrid Universal

### DISCUSSION THREAD

**Discussion by hwlv** [ghiscoding/slickgrid-universal#1715] [General]
Description:
Currently, setCellCssStyles allows setting CSS styles for rows or columns, but it does not seem to support applying styles to specific cells. I need the ability to apply custom styles to individual cells based on specific conditions (e.g., cell value or other row/column-specific data). This feature would be highly beneficial for scenarios like highlighting errors or dynamically formatting cells based on data.

Expected Behavior:
Provide a method or enhance the existing setCellCssStyles to target individual cells, not just entire rows or columns.

**Comment by ghiscoding** [comment]
That's exactly what [Example 11](https://ghiscoding.github.io/slickgrid-universal/#/example11) and [Example 12](https://ghiscoding.github.io/slickgrid-universal/#/example12) do after editing a cell, any edited cell(s) will change color. You ask a lot of questions in here, however I wish that you could consider asking these kind of questions on Stack Overflow, it's a better platform for any kind of questions and I also answer questions there too. For example there's already this SO Question [Add Css style to edited cell in slickgrid](https://stackoverflow.com/questions/14514999/add-css-style-to-edited-cell-in-slickgrid) that also seems to fit your question. 

The `setCellCssStyles` is really the correct function to use, you need to provide your row in the hash key as shown below from Example 12

https://github.com/ghiscoding/slickgrid-universal/blob/a0a595e919c4017568ff2f55f2b68d49c10a868e/examples/vite-demo-vanilla-bundle/src/examples/example12.ts#L689-L699
